frontend/src/components/MainNav.styles.js

```diff
@@ -1,17 +1,19 @@
 ﻿import styled from 'styled-components';
 
-export const NavBar = styled.nav`
+export const NavBar = styled.div`
```

Comment:
I think `nav` was more accurate here than `div`. Why the change?

Reply:
Lets delete this change